conf/carbon.conf.example

```diff
@@ -659,6 +667,10 @@ MAX_AGGREGATION_INTERVALS = 5
 # In order to turn off logging of successful connections for the line
 # receiver, set this to False
 # LOG_LISTENER_CONN_SUCCESS = True
+#
+# To turn off logging of lost connections for the line receiver set this to
+# False
```

Comment:
Since the default is `False`, this should probably say "To turn on logging of lost connections, set this to True"

Reply:
I'm happy to submit another MR with the change if you think it's important.  I personally think it's clear enough but I'm happy to do whatever the maintainers think is best.